src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import Portal from '../Portal';
+import { BackDrop, Content, Wrapper } from './styles';
+import { DrawerProps } from './types';
```

Comment:
Please check this ESLint error.

note: You might have to run `npm ci` again to run eslint server.